lib/helpers/priceHelpers.ts

```diff
@@ -44,3 +44,13 @@ export function formatIntPrice(price: number): string {
 function isPriceValid(price: number) {
   return !Number.isNaN(price) && Number.isFinite(price) && price >= 0;
 }
+
+/**
+ * Get suggested used price from a new price
+ *
```

Comment:
Maybe add a short comment as to how the suggestion is generated? The code is kind of opaque (like I think it's doing price * 0.6 but I'm not sure)